backend/YouTubeV2.Application/Services/JwtFeatures/JwtHandler.cs

```diff
@@ -36,7 +37,8 @@ private async Task<ICollection<Claim>> GetClaimsAsync(User user)
         {
             var claims = new List<Claim>
             {
-                new Claim(ClaimTypes.Name, user.Email)
+                new Claim(ClaimTypes.Name, user.Email),
```

Comment:
maybe add role to claims so we can quickly check if user should have access to endpoint

Reply:
roles are added to claims just below your selection